docker/ci.override.yml

```diff
@@ -0,0 +1,38 @@
+version: '3.8'
+
+services:
+  postgres:
+    env_file:
+      - env_file_postgres_template
+
+  uwsgi:
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile
+    image: honeynet/greedybear:ci
+    env_file:
+      - env_file_app_ci
+  
+  nginx:
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile_nginx
+    image: honeynet/greedybear:ci
```

Comment:
this name should be different from the other one...the dockerfiles are different

Reply:
call it `honeynet/greedybear_nginx:ci`

Reply:
Thanks a lot for help!!